src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/personalisation/Personalisation.java

```diff
@@ -558,8 +589,13 @@ Map<String, Object> setEvidenceProcessingAddress(Map<String, Object> personalisa
             personalisation.put(REGIONAL_OFFICE_POSTCODE_LITERAL, rpc.getPostcode());
         }
 
-        personalisation.put(PHONE_NUMBER_WELSH, evidenceProperties.getAddress().getTelephoneWelsh());
-        personalisation.put(PHONE_NUMBER, determinePhoneNumber(rpc));
+        if (!Objects.equals(ccdResponse.getIsScottishCase(), "Yes") && ccdResponse.isIbcCase()) {
```

Comment:
`"Yes".equals(ccdResponse.getIsScottishCase())` don't we have a helper function for this already?